polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -273,15 +322,55 @@ class FrameInner extends PureComponent<CombinedProps, State> {
           {skipMarkup}
           {topBarMarkup}
           {navigationMarkup}
-          {contextualSaveBarMarkup}
+          <UseFeatures>
+            {({polarisSummerEditions2023}) => {
+              return !polarisSummerEditions2023 ? (
+                <CSSAnimation
+                  in={showContextualSaveBar}
+                  className={styles.ContextualSaveBar}
+                  type="fade"
+                >
+                  <ContextualSaveBar {...this.contextualSaveBar} />
+                </CSSAnimation>
+              ) : (
+                <></>
+              );
+            }}
```

Comment:
nit. Based on [no-negated-condition](https://eslint.org/docs/latest/rules/no-negated-condition) rule, it's often preferred to avoid ternaries that start with a  negated condition